Dockerfile

```diff
@@ -336,6 +369,7 @@ COPY --link --from=terragrunt /usr/local/bin/terragrunt /usr/bin/
 COPY --link --from=terragrunt /bin/terraform /usr/bin/
 COPY --link --from=kics /app/bin/kics /usr/bin/
 COPY --from=kics /app/bin/assets /opt/kics/assets/
+COPY --from=cargo /bin/* /usr/bin/
```

Comment:
When I added the COPY --link, I wasn't sure what would happen if there are multiple files copied and there are some existing files already in the destination folder. If you would've used copy with link here anyways, since you are using it in your configurations, go for it and change it ;)

Reply:
Done, I also tweaked the deploy-DEV.yml to (hopefully) properly build and start the test of the image (I haven't touched the tests yet)